tests/ClientTest.php

```diff
@@ -210,6 +207,24 @@ public function join($timeout = null)
     }
 }
 
+class Dummy_Serializer implements \Raven\SerializerInterface
```

Comment:
I know the scope of that test is to check that the serializer is correctly set, but there is no need to check the value returned after serializing a message to do it. You should create a mock, pass it in place of the current `Dummy_Serializer` object and assert that its `serialize` method is called when capturing an exception/message. Nothing more, nothing else. No checks on how the sent message is serialized itself